arelle/plugin/validate/ESEF/__init__.py

```diff
@@ -148,18 +169,22 @@ def validateXbrlStart(val, parameters=None, *args, **kwargs):
     val.consolidated = not val.unconsolidated
     val.authority = None
     if parameters:
-        p = parameters.get(qname("authority",noPrefixIsNoNamespace=True))
+        p = parameters.get(qname("authority",noPrefixIsNoNamespace=True)) #  type: ignore[no-untyped-call]
         if p and len(p) == 2 and p[1] not in ("null", "None", None):
             v = p[1] # formula dialog and cmd line formula parameters may need type conversion
             val.authority = v
-    
+
     authorityValidations = loadAuthorityValidations(val.modelXbrl)
+    # loadAuthorityValidations returns either a list or a dict but in this context, we expect a dict.
+    # By using cast, we let mypy know that a list is _not_ expected here.
+    authorityValidations = cast(Dict[Any, Any], authorityValidations)
```

Comment:
`Typing.Dict` [is deprecated](https://docs.python.org/3/library/typing.html#typing.Dict). Use `dict` instead:
```suggestion
    authorityValidations = cast(dict[Any, Any], authorityValidations)
```